spec/controllers/rebate_forms_controller_spec.rb

```diff
@@ -8,6 +8,16 @@
   let(:fields) { { 'dependants' => '0', 'full_name' => 'bob', 'income' => '10000' } }
   let(:property) { FactoryBot.create :property_with_rates, rating_year: ENV['YEAR'] }
 
+  before do
```

Comment:
These tests are a good argument for moving the JWT code into a service which will handle this stuff for you